### ISSUE TITLE
Bug: pageContext.queryClient isn't defined on the client-side

### ISSUE THREAD

**Issue by flyerH** [vikejs/vike-vue#163]
![4dcb0de073caab49c808e338f2ea651d](https://github.com/user-attachments/assets/806b2827-ce73-4a51-85d9-ef4e6d958a48)

This is a mini repo
[https://stackblitz.com/edit/vitejs-vite-sysrfj?file=package.json](https://stackblitz.com/edit/vitejs-vite-sysrfj?file=package.json)

1. Add /id such as /1 to the end of the link on the preview page on the right
2. Click on the hyperlink
<img src="https://github.com/user-attachments/assets/379fc94d-0804-4d2c-8f27-213a825338ef" width=50%>

**Comment by brillout** [comment]
Are you using the latest version of all dependencies? Can you publish the reproduction on GitHub?

**Comment by flyerH** [comment]
> Are you using the latest version of all dependencies? Can you publish the reproduction on GitHub?

Yes, I have upgraded to the latest version, this is repo on Github [https://github.com/flyerH/vike-vue-example](https://github.com/flyerH/vike-vue-example)  
Run and open `http://localhost:3000/1234`

**Comment by flyerH** [comment]
And I found this modify can fix problem, but I don't know if it will cause new problem.

```diff
// https://github.com/vikejs/vike-vue/blob/main/packages/vike-vue-query/renderer/hydrateVueQuery.ts
- const hydrateVueQuery = ({ queryClient, fromHtmlRenderer, isHydration }) => {
+ const hydrateVueQuery = (params) => {
+   const { fromHtmlRenderer, isHydration } = params;
+   const queryClient = 'queryClient' in params ? params.queryClient : null;
    if (!isHydration) {
        return;
    }
    const { vueQueryInitialState } = fromHtmlRenderer;
    if (!queryClient || !vueQueryInitialState) {
        // happens if SSR is off
        return;
    }
    hydrate(queryClient, vueQueryInitialState);
};
```

**Comment by brillout** [comment]
I went for your fix and pre-released it as `vike-vue-query@0.1.1-commit-4934997`.

Let's see if you still have issues.

(Btw. in case your company is up for it [we're looking for sponsors](https://github.com/vikejs/vike/issues/1350).)

**Comment by flyerH** [comment]
The problem is solved, thanks~

**Comment by flyerH** [comment]
Hello, can you please publish a release version of vike-vue-query?

**Comment by brillout** [comment]
Released `vike-vue-query@0.1.2`.
